src/orion/benchmark/task/profet/forrester.py

```diff
@@ -0,0 +1,65 @@
+""" Simulated Task consisting in training a Random Forrest predictor.
+
+The task is created using the Profet algorithm:
+
+```
+@misc{klein2019metasurrogate,
+      title={Meta-Surrogate Benchmarking for Hyperparameter Optimization}, 
+      author={Aaron Klein and Zhenwen Dai and Frank Hutter and Neil Lawrence and Javier Gonzalez},
+      year={2019},
+      eprint={1905.12982},
+      archivePrefix={arXiv},
+      primaryClass={cs.LG}
+}
+```
+"""
+from dataclasses import dataclass
+from logging import getLogger as get_logger
+from pathlib import Path
+from typing import Dict, Union
+
+import torch
+from orion.benchmark.task.profet.profet_task import MetaModelTrainingConfig, ProfetTask
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class ForresterTaskHParams:
+    """Hyper-Parameters of a Simulated Task consisting in training a Random Forrest predictor."""
+
+    # TODO (@lebrice): This space is supposedly not correct. Need to look at
```

Comment:
Issue? Or should be solve in this PR?

Reply:
I'd vote for this to be handled in another PR

Reply:
Sounds good! Can you please create an issue and remove the TODO note. Thanks!

Reply:
I've actually reconsidered, this isn't an issue, it's just that the OpenML and Profet Forrester tasks have different spaces.